Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM node:10-alpine
 WORKDIR /usr/src/app
 
+RUN apk add git
```

Comment:
Please add --no-cache --virtual to the add command. The virtual flag ensure that the delete command uninstalles git and all its dependencies

Reply:
Done